pkg/yurthub/cachemanager/cache_manager.go

```diff
@@ -561,6 +606,11 @@ func isCreate(ctx context.Context) bool {
 // 4. csr resource request
 func (cm *cacheManager) CanCacheFor(req *http.Request) bool {
 	ctx := req.Context()
+
+	if req.URL.Path == "/version" {
```

Comment:
Does it cover the case if we add some parmeters in the url such as `/version?timeout=10s`

Reply:
Yes, Path only refers to "/version" except for the get request arguments